plugins/utils/flash.nix

```diff
@@ -11,6 +11,7 @@ helpers.neovim-plugin.mkNeovimPlugin config {
   originalName = "flash.nvim";
   defaultPackage = pkgs.vimPlugins.flash-nvim;
 
+  skipConfigAttrs = true;
```

Comment:
I don't understand why this is needed? I can't see anything config-related in the flash module. Do you mind clarifying?

Reply:
The flash plugin has a renamed option from `config` to `settings.config`

Reply:
Ah, I see. That's unfortunate!

I wonder if `config` is the best name for the new option anyway... It's already an overloaded word in nixos modules, since we have a config attr in the module output, plus a `config` module arg, plus modules in general usually relate to _configuration_...

Not saying we should choose a different name _just because_ of this conflict though. Only if we actually think of a better name.

Reply:
I was wondering this, as well... but I don't know what else you'd choose.

Reply:
Naming things isn't my strong suit, if you have better ideas don't hesitate!

Reply:
Perhaps `luaConfig`,  `luaText`, or even just `lua`?